### PR TITLE
feat(api): decrease plunger acceleration

### PR DESCRIPTION
**Pull request by andySigler**
## overview

Decreases plunger acceleration to allow smoother motions, and puts less strain on the internal seal (o-ring) when changing direction.

In addition to lowering the default acceleration, this PR also adds methods for setting and pushing/popping acceleration, similar to how the driver sets speed.

Finally, this PR also decreases the plunger speed to 5mm/sec while dropping tips, to increase torque.

All the following changes have been tested and approved by @Carlos-fernandez

### CODE REVIEW

File: api/src/opentrons/legacy_api/instruments/pipette.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 }
 
 DROP_TIP_RELEASE_DISTANCE = 20
+DROP_TIP_SPEED = 5
```
Comment (sfoster1) [inline api/src/opentrons/legacy_api/instruments/pipette.py]: Instead of being a hardcoded number, this should be added to the pipetteModelSpecs so it can be one of the things that can be overridden by the user in future work.

File: api/src/opentrons/legacy_api/instruments/pipette.py [hunk 2]
```diff
@@ -683,6 +684,10 @@ def blow_out(self, location=None):
 
         self.move_to(location)
         self.instrument_actuator.set_active_current(self._plunger_current)
+        self.robot.poses = self.instrument_actuator.move(
```
Comment (sfoster1) [inline api/src/opentrons/legacy_api/instruments/pipette.py]: What's this change about? It seems unrelated

Comment (andySigler) [reply api/src/opentrons/legacy_api/instruments/pipette.py]: you're right! it was left over from some other change that has since been deleted. Removing now...

**Comment by sfoster1** [comment]
should be able to set the speed in the new api here:
https://github.com/Opentrons/opentrons/blob/c5b375c360d3cbedf8971c91751e5ad595a9cf41/api/src/opentrons/hardware_control/__init__.py#L941

add `speed=instr.config.drop_tip_speed` as a kwarg

File: api/src/opentrons/hardware_control/__init__.py [hunk 3]
```diff
@@ -938,7 +938,8 @@ def _plunger_position(self, instr: Pipette, ul: float,
         await self._move_plunger(mount, bottom)
         self._backend.set_active_current(plunger_ax,
                                          instr.config.drop_tip_current)
-        await self._move_plunger(mount, droptip)
+        await self._move_plunger(
+            mount, droptip, speed=instr.config.drop_tip_speed)
```
Comment (andySigler) [inline api/src/opentrons/hardware_control/__init__.py]: @sfoster1 like this?

Comment (sfoster1) [reply api/src/opentrons/hardware_control/__init__.py]: yup
